rusoto/core/src/lib.rs

```diff
@@ -52,13 +53,17 @@ extern crate base64;
 pub extern crate rusoto_credential as credential;
 extern crate serde;
 extern crate time;
+extern crate tokio_core;
 extern crate url;
 extern crate xml;
 
+mod future;
```

Comment:
If we're `pub use future::RusotoFuture` why not make this a: `pub mod` as well?

Reply:
I didn't make it a pub mod, since the only thing it exports is `RusotoFuture`, which is already exported on the top level. Happy to expose it if you want me to.